server/src/model/chapters.go

```diff
@@ -63,10 +63,13 @@ type ChapterWithToken struct {
 }
 
 type Organizer struct {
-	Name     string
-	Email    string
-	Phone    string
-	Facebook string
+	Name      string
+	Email     string
+	Phone     string
+	Facebook  string
+	Instagram string
+	Twitter   string
+	Website   string
```

Comment:
when reading organizers out of the db that don't already have these fields set in the json, i assume it will be fine and just return go's nil values (empty string)? my go is rusty....

Reply:
yeah i think so. i was able to add fields here after creating test organizers and they loaded fine.